src/web/app/pages-instructor/instructor-session-edit-page/instructor-session-edit-page.component.ts

```diff
@@ -556,6 +557,15 @@ export class InstructorSessionEditPageComponent extends InstructorSessionBasePag
   private moveQuestionForm(originalPosition: number, newPosition: number): void {
     this.questionEditFormModels.splice(newPosition, 0,
         this.questionEditFormModels.splice(originalPosition, 1)[0]);
+
+    // all expanded questions that were moved upwards must be re-expanded to reload rich text editor
+    const start: number = Math.min(originalPosition, newPosition);
+    const movedExpandedQuestions: QuestionEditFormModel[] = this.questionEditFormModels
+      .slice(start, newPosition + 1)
+      .filter((model: QuestionEditFormModel) => !model.isCollapsed);
+    movedExpandedQuestions.forEach((model: QuestionEditFormModel) => model.isCollapsed = true);
+    this.changeDetectorRef.detectChanges();
+    movedExpandedQuestions.forEach((model: QuestionEditFormModel) => model.isCollapsed = false);
```

Comment:
This will make all modals previously collapsed to be expanded. Only those which are previously expanded should be re-expanded.

Reply:
@wkurniawan07  I have filtered to only those that are not collapsed so those that are originally collapsed will not be affected. (line 565)

Reply:
Ah right, I missed that.